### DISCUSSION TITLE
New Source: Redis

### DISCUSSION THREAD

**Discussion by sherifnada** [airbytehq/airbyte#35264] [New Connector Request]
## Tell us about the new integration you’d like to have
I'd like to be able to pull data from a Redis instance. 

## Describe the alternative you are considering or using

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1200367912513076/1200368165389619) by [Unito](https://www.unito.io)

**Comment by zlwzlwzlw** [comment]
Source Redis does support it now?

[a comment by an automated account is not shown]
